addons/binding/org.openhab.binding.mihome/src/main/java/org/openhab/binding/mihome/internal/ColorUtil.java

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.mihome.internal;
+
+/**
+ * Color utilities.
+ *
+ * @author paulkaplan
```

Comment:
Who is this, is he okay with contributing this here, on what license did he make his original contribution?